en/08_Changelogs/6.0.0.md

```diff
@@ -916,12 +917,21 @@ SilverStripe\ORM\Connect\MySQLDatabase:
 
 ### `DBDecimal` default value {#dbdecimal-default-value}
 
-Previously if an invalid default value was provided for a [`DBDecimal`](api:SilverStripe\ORM\FieldType\DBDecimal) database column, it would silently set the defalt value to `0`. This will now throw an exception instead, so that you're aware your configured value is invalid and can correct it.
+Previously if an invalid default value was provided for a [`DBDecimal`](api:SilverStripe\ORM\FieldType\DBDecimal) database column, it would silently set the default value to `0`. This will now throw an exception instead, so that you're aware your configured value is invalid and can correct it.
 
 ### `RedirectorPage` validation
 
 [`RedirectorPage`](api:SilverStripe\CMS\Model\RedirectorPage) now uses the [`Url` constraint](https://symfony.com/doc/current/reference/constraints/Url.html) from `symfony/validator` to validate the `ExternalURL` field. It will no longer add `http://` to the start of URLs for you if you're missing a protocol - instead, a validation error message will be displayed.
 
+### Change JS MIME type & remove default `type` attribute in `<script>` tags {#change-js-mime-type}
+
+Updated the MIME type for JavaScript from `"application/javascript"` to `"text/javascript"`. Additionally, the `type` attribute has been omitted from `<script>` tags since it defaults to `"text/javascript"` in HTML5, and it is encouraged to omit it instead of redundantly setting it.
```

Comment:
```suggestion
We've updated the MIME type for JavaScript from `"application/javascript"` to `"text/javascript"`. Additionally, the `type` attribute has been omitted from any `<script>` tags generated by Silverstripe CMS itself (e.g. in the Requirements API). The most up-to-date RFC says to use `"text/javascript"` in HTML5. Since modern browsers will default to that type when one isn't explicitly declared, it is generally encouraged to omit it instead of redundantly setting it.
```